### PR TITLE
weaponSelect CBA Setting

### PR DESCRIPTION
**Pull request by commy2**
This is an easy one.

### CODE REVIEW

**Comment by PabstMirror** [comment]
Not really related to this PR, just something I saw in testing
```
[
    "test1", "CHECKBOX",
    ["test1", LSTRING(SettingDisplayTextDesc)],
     localize "STR_ACE_common_ACEKeybindCategoryWeapons"
] call CBA_settings_fnc_init;

[
    "test2", "CHECKBOX",
    ["test2"],
    "STR_ACE_common_ACEKeybindCategoryWeapons"
] call CBA_settings_fnc_init;
```

Running both of these produces weird results.
It opens on the localized category with only test1
Category lists both "ACE3 Weapons" and "STR_ACE_common_ACEKeybindCategoryWeapons"
The "STR" has nothing in it, reselecting the localized one then shows both options.

I'm thinking we should just have CBA localize categories on init so there's no problems on category filtering?

**Comment by commy2** [comment]
lgtm, can't approve what I opened.

File: addons/weaponselect/initSettings.sqf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// CBA Settings [ADDON: ace_weaponselect]:
+
+[QGVAR(displayText), "CHECKBOX",
+[LSTRING(SettingDisplayTextName), LSTRING(SettingDisplayTextDesc)],
+"ACE Uncategorized",
```
Comment (SilentSpike) [inline addons/weaponselect/initSettings.sqf]: Is there some sort of inventory category this would be more suited to?

**Comment by commy2** [comment]
UI

**Comment by PabstMirror** [comment]
Requires #5752

**Pull request merged**